docs/contributor/benchmarks/otlp-logs-validation.md

```diff
@@ -0,0 +1,312 @@
+# OTel LogPipeline set-up validation
+
+This file documents the process of validating the whole LogPipeline with OTLP output flow. It starts by defining the setup, that consists of the manually deployed log agent, the already-implemented log gateway, and log generators using flog.
+
+The scope is to performance test the agent, observing the resulting values, in terms of throughput, resource consumption, reaction to backpressure, etc. And compare it to the previous FluentBit-based setup.
+
+
+
+## 1. Set-up configuration steps
+
+### With Helm
+
+``` bash
+k apply -f telemetry-manager/config/samples/operator_v1alpha1_telemetry.yaml
+
+// Execute knowledge-hub/scripts/create_cls_log_pipeline.sh with the corresponding environment variables 
+
+helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+
+helm install -n kyma-system logging open-telemetry/opentelemetry-collector -f telemetry-manager/docs/contributor/pocs/assets/otel-log-agent-values.yaml
+```
+
+### Manual
+
+``` bash
+k apply -f telemetry-manager/config/samples/operator_v1alpha1_telemetry.yaml
+
+// Execute knowledge-hub/scripts/create_cls_log_pipeline.sh with the corresponding environment variables 
+
+k apply -f ./otlp-logs-validation.yaml
+```
+
+
+
+## 2. Resulting Resources
+
+### Agent ConfigMap (OTel Config)
```

Comment:
This doesn't need to be a subheadline. Much simpler to put the two resources into an unordered list.

```suggestion
- Log Agent ConfigMap (OTel Config)
- Log Agent DaemonSet

```